src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -18,9 +18,23 @@ public static partial class Certificates
         {
             private const string CertificatePassword = "testcertificate";
             private const string TestDataFolder = "TestData";
-            private static Mutex m = new Mutex(false, "Global\\CoreFXTest.Configuration.Certificates.LoadPfxCertificate");
+
+            private static Mutex m;
             private const int MutexTimeout = 120 * 1000;
 
+            static Certificates()
+            {
+                if (PlatformDetection.IsUap)
```

Comment:
Uap and UapAot should behave the same way.  You're probably seeing #18760 - for some reason UapAot is not ran in an AppContainer and so it won't show the same behavior.